lib/shopify_app/configuration.rb

```diff
@@ -83,7 +83,17 @@ def shop_access_scopes_strategy
       ShopifyApp::AccessScopes::ShopStrategy
     end
 
+    def user_access_scopes_strategy=(class_name)
+      unless class_name.is_a?(String)
```

Comment:
Using a string to be consistent with `user_session_repository` and `shop_session_repository`.